en/docs/integration-guides/usecases/datamapper/read-csv-file-and-transform-to-xml-file.md

```diff
@@ -46,19 +46,19 @@ S003,P-2003,1,89.50
 ```
 
 ### What you’ll configure in the UI
-- **Source (CSV)**: delimiter `,` header enabled, fields `order_id, sku, qty, price`.
-- **Target (XML)**: root `Orders`, repeating child `Row` with children `index`, `order_id`, `sku`, `qty`, `price` — all as **elements**.
-- **Row loop**: create one `Row` per CSV data row.
-- **Index**: bind `index` to the mapper’s row position (to ensure 0-based numbering).
-- **Type handling** (optional but recommended): cast `qty` to `int`, `price` to `decimal`; set defaults for missing/invalid values.
-- **File I/O**: 
+- Source (CSV): delimiter `,` header enabled, fields `order_id, sku, qty, price`.
+- Target (XML): root `Orders`, repeating child `Row` with children `index`, `order_id`, `sku`, `qty`, `price` — all as elements.
+- Row loop: create one `Row` per CSV data row.
+- Index: bind `index` to the mapper’s row position (to ensure 0-based numbering).
+- Type handling (optional but recommended): cast `qty` to `int`, `price` to `decimal`; set defaults for missing/invalid values.
+- File I/O: 
   - Inbound: read `customer_order_details.csv` from `input/`.
   - Outbound: write `customer_order_details.xml` to `output/`.
 
 ### Why this use case
```

Comment:
This should be part of the first sub topic under `Overview`? Then user will understand what he is going to do in the first place. some of the points are repetitive points in the overview section. So instead of having this as seperate sub section, can't we merge it to the first para udner overview?

Reply:
Overview is for the datamapper usecases.. not for the individual usecase. There are more use cases coming in. Also there are other categories as well. (file integration, event integration.. etc)